src/config.ts

```diff
@@ -1,803 +1,1213 @@
-const envalid = require("envalid");
-const pickBy = require("lodash/pickBy");
+import * as envalid from "envalid";
```

Comment:
oh sorry, i thought a preprocessor was set up to ensure module imports for the stuff still in js. it seems like only `logger.js`, `src/lib/index.js`, `knex.ts`, `crypto.js`, and `gs.ts` use this config though so i can take responsibility for typing the js stuff if there's trouble in the future. 

> > hm yeah i think i might have ran prettier weirdly. i do have format-on-save on but lint-staged was complaining so i did a `prettier --write` at the repo root. it changed every `migrations/...js` file though
> 
> That's odd 🤔 seems like it may need to have the config file specified explicitly with `--config ./.prettierrc.js`.
> 
> What errors was lint-staged complaining about that it wouldn't just fix? And I guess that your editor wasn't complaining about either?

hm i don't really know what i was doing before (maybe i was using npx to run lint + prettier)? but i seem to have randomly-done-things until it was fine again lol

Reply:
oh sorry, i thought a preprocessor was set up to ensure module imports for the stuff still in js. it seems like only `logger.js`, `src/lib/index.js`, `knex.ts`, `crypto.js`, and `gs.ts` use this config though so i can take responsibility for typing the js stuff if there's trouble in the future.